packages/owlbot-bootstrapper/package.json

```diff
@@ -18,13 +18,9 @@
     "build/src"
   ],
   "scripts": {
-    "compile": "tsc -p .",
-    "start": "probot run ./build/src/googleapis-bootstrapper.js",
-    "start:local": "node ./build/common-container/common-container.js",
-    "pretest": "npm run compile",
-    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test",
-    "fix": "gts fix",
-    "lint": "gts check"
+    "test": "cd cli; npm t; cd ../app; npm t; cd ../common-container; npm t",
+    "fix": "cd cli; npm run fix; cd ../app; npm run fix; cd ../common-container; npm run fix",
+    "lint": "cd cli; npm run lint; cd ../app; npm run lint; cd ../common-container; npm run lint"
```

Comment:
nit: this can be done in a follow up, but if you do run the test from this parent folder, the `;` will make it so each test runs regardless of if the previous command passed or failed. If the `cli` tests fail, then it will continue. If the `common-container` tests pass, then it will report success regardless of if `cli` or `app` tests failed